pkg/encoding/hex/hex.go

```diff
@@ -0,0 +1,41 @@
+package hex
```

Comment:
Not sure why this PR includes the implementation of Hex, was that an issue?

Reply:
I need it for the test :)